src/main/target/common_fc_pre.h

```diff
@@ -126,6 +126,7 @@
 #define VTX_TRAMP
 #define USE_CAMERA_CONTROL
 #define USE_HUFFMAN
+#define USE_COPY_PROFILE_CMS_MENU
```

Comment:
Why do we need a separate define for this?

Reply:
As commented earlier the CMS code for coping profiles is categorized as a 'luxury' and can be disabled when a target overflows flash space. The define that I used initially is from another PR that is not yet merged, as this PR has the go with BF 3.2, i've made a separate define for it.

Reply:
Ah, fair enough. We can always rename it back to make it easier to trim  down the CMS when the other PR is merged. What's the other PR's number?

Reply:
This is PR's that I referred to: https://github.com/betaflight/betaflight/pull/3601